.fixtures.yml

```diff
@@ -1,5 +1,6 @@
 fixtures:
   repositories:
     stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+    debug: 'https://github.com/nwops/puppet-debug.git'
```

Comment:
I don't understand why this is in a change related to adding sensitive type

Reply:
Debug is like pry for puppet   it is only a dev dependency.